config/config.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+'use strict';
+
+const fs = require('fs');
+
+const TARGET = Object.freeze({
+  ORIGIN: 'origin',
+  AUXILIARY: 'auxiliary',
+});
+
+const readConfigFile = (filePath) => {
+  const configFile = fs.readFileSync(filePath);
+  const config = JSON.parse(configFile);
+
+  return config;
+};
+
+const readAnchorConfig = (configFile, target, anchorCmd) => {
+  const regexp = new RegExp(`(${TARGET.ORIGIN}|${TARGET.AUXILIARY})`);
+  if (target.match(regexp) === null) {
```

Comment:
checks for valid target

Reply:
Regex seems a bit overkill here.

We could also do `if(Object.values(TARGET).includes(target)) {`